### PR TITLE
Added additional information on how to find the executablePath.

### PR DESCRIPTION
**Pull request by Reezaali**

### CODE REVIEW

File: microsoft-edge/puppeteer.md [hunk 1]
```diff
@@ -43,35 +45,88 @@ const puppeteer = require('puppeteer-core');
...
-> The **example.png** file produced by **example.js**
+> The `example.png` file produced by `example.js`  
 > ![The example.png file produced by example.js](./media/puppeteer-example.png)  
+-->  
+
+This is just a simple example of the automation and testing scenarios enabled by Puppeteer and `puppeteer-core`.  For more information about Puppeteer and how it works, see [Puppeteer][PuppeteerMain].  
+
+## Feedback  
+
+The Edge Developer team is eager to hear your feedback about using Puppeteer, `puppeteer-core`, and Microsoft Edge!  Use the **Feedback** icon in the Microsoft Edge DevTools or tweet [@EdgeDevTools][TwitterIntentTweetEdgedevtools] to let the Microsoft Edge team know what you think.
```
Comment (Reezaali) [inline microsoft-edge/puppeteer.md:82]:
```suggestion
## Send Feedback  

The Edge Developer team is eager to hear your feedback about using Puppeteer, `puppeteer-core`, and Microsoft Edge.  Use the **Send Feedback** icon in the Microsoft Edge DevTools or tweet [@EdgeDevTools][TwitterIntentTweetEdgedevtools] to let the Microsoft Edge team know what you think.  
```

File: microsoft-edge/puppeteer.md [hunk 2]
```diff
@@ -43,35 +45,88 @@ const puppeteer = require('puppeteer-core');
...
+   Figure 2:  The **Feedback** icon in the Microsoft Edge DevTools  
+:::image-end:::  
+
+<!--  
+> ##### Figure 2  
+> The **Feedback** icon in the Microsoft Edge DevTools  
+> ![The Feedback icon in the Microsoft Edge DevTools](./devtools-guide-chromium/media/devtools-feedback.png)  
+-->  
+
+## See also
```
Comment (Reezaali) [inline microsoft-edge/puppeteer.md:95]: Since some of these links are mentioned in the article, can we reduce the length of the See Also list?

Comment (jm-trd-ms) [reply microsoft-edge/puppeteer.md]: Removed redundant links.

**Comment by jm-trd-ms** [comment]
#sign-off and merge.

**Pull request merged**